### ISSUE TITLE
NPC Placement Problems (not a priority) 

### ISSUE THREAD

**Issue by ShredzAlot** [henbagle/LE1CommunityPatch#147]
**A small list of placement problems for a couple of npcs. Not groundbreaking.**

It's hard to tell in this shot but the guy Powell, his hands are clipping into the crate infront of him. It's not a big deal becasue its such a quick convo but when the camera switches angles from time to time you can notice it.
![Mass Effect 2022-05-01 6_01_07 PM](https://user-images.githubusercontent.com/95461294/170888034-06198d06-63e1-455c-8e19-22053e9b7c82.png)

This guy is always obvious to me when walking to the markets.
![ScreenShot00009](https://user-images.githubusercontent.com/95461294/170888153-4e65690b-9cb9-4443-8324-aebbbc07c6ec.jpg)

The woman on Feros but she doesn't stay like that for the entire time. Might just have to live with this one
![ScreenShot00031](https://user-images.githubusercontent.com/95461294/170888157-9e58b800-c98e-46ff-9898-226922485f6d.jpg)

It will fix itself only after you talk to her.
![ScreenShot00032](https://user-images.githubusercontent.com/95461294/170888161-11541efa-cb97-4ceb-be0e-de10cf21976c.jpg)
